tests/cbmc/proofs/s2n_stuffer_init/s2n_stuffer_init_harness.c

```diff
@@ -15,10 +15,15 @@
 
 #include "api/s2n.h"
 #include "stuffer/s2n_stuffer.h"
+#include <assert.h>
 #include <cbmc_proof/proof_allocators.h>
+#include <cbmc_proof/make_common_datastructures.h>
 
 void s2n_stuffer_init_harness() {
-  struct s2n_stuffer *stuffer = malloc(sizeof(*stuffer));
-  struct s2n_blob *in = malloc(sizeof(*in));
-  int result = s2n_stuffer_init(stuffer, in);
+  struct s2n_stuffer *stuffer = can_fail_malloc(sizeof(*stuffer));
+  struct s2n_blob* in = cbmc_allocate_s2n_blob();
+  if (s2n_stuffer_init(stuffer, in) == S2N_SUCCESS){
+    assert(s2n_stuffer_is_valid(stuffer));
+    assert(s2n_blob_is_valid(in));
```

Comment:
Shouldn't we be asserting that `in` is valid prior to calling `s2n_stuffer_init(stuffer, in)` on line 25?

Reply:
Good question.  This comes down to style.  In this case, `s2n_stuffer_init` will reject an invalid `s2n_blob` with an error code, so we don't need to also assume it.  What would make sense is to have a check that the `blob` wasn't modified in any way during the execution of the function.  I can add that.